### ISSUE TITLE
Modify the `aria-label` for device lookup "Select" button to contain device name

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5133]
## Area or Component
`/results/upload/submit/code-lookup`

## Quick Summary
The "Select" button that appears when searching for a device with multiple options does not have a descriptive label for which device it is pertaining to.

### How to Replicate
Search "Abbott" or a device with multiple search results in the device code lookup search box. Use voice over to navigate to a device's "Select" button. You will hear no context for which device this is for.

## Accessibility Violation
[Name: The element's name is missing or incorrect](https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect)

## Action Requested
Add the device's name to the `aria-label` for "Select" button.

## Screenshots (optional)
![Screen Shot 2023-01-25 at 21.15.27.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/93ee3974-af32-47d1-9d2b-356666d22b50)
## Source Code (optional)

[which file or line is the issue attached to?]: #

## Original Deque Issue
